controller/share_manager_controller.go

```diff
@@ -740,8 +741,22 @@ func (c *ShareManagerController) createShareManagerPod(sm *longhorn.ShareManager
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to create pod for share manager %v", sm.Name)
 	}
+
+	obj, err := datastore.VerifyCreation(pod.Name, "pod", func(name string) (runtime.Object, error) {
+		return c.ds.GetPodRO(pod.Namespace, name)
+	})
```

Comment:
Similarly, why execute this outside of `ds.CreatePod`?

Reply:
I updated the PR. Can you re-check it? Thank you.

Reply:
I remove this commit and it will be handled in https://github.com/longhorn/longhorn/issues/5233.